Cannot install ballista-scheduler

**Describe the bug**
Getting error when installing `ballista-scheduler`

**To Reproduce**
Run `cargo install ballista-scheduler`

**Expected behavior**
Install successfully

**Additional context**
```
error[E0599]: no function or associated item named `from_str_unchecked` found for struct `proc_macro2::Literal` in the current scope
   --> C:\Users\selva\.cargo\registry\src\github.com-1ecc6299db9ec823\quote-1.0.11\src\runtime.rs:301:41
    |
301 |         let literal = unsafe { Literal::from_str_unchecked(repr) };
    |                                         ^^^^^^^^^^^^^^^^^^ function or associated item not found in `proc_macro2::Literal`

error[E0599]: no function or associated item named `from_str_unchecked` found for struct `proc_macro2::Literal` in the current scope
   --> C:\Users\selva\.cargo\registry\src\github.com-1ecc6299db9ec823\quote-1.0.11\src\runtime.rs:310:45
    |
310 |         let mut literal = unsafe { Literal::from_str_unchecked(repr) };
    |                                             ^^^^^^^^^^^^^^^^^^ function or associated item not found in `proc_macro2::Literal`

   Compiling twoway v0.1.8
For more information about this error, try `rustc --explain E0599`.

--- a/ballista/rust/client/README.md
+++ b/ballista/rust/client/README.md
@@ -49,8 +49,8 @@ A simple way to start a local cluster for testing purposes is to use cargo to in
 the scheduler and executor crates.
 
 ```bash
-cargo install ballista-scheduler
-cargo install ballista-executor
+cargo install --locked ballista-scheduler
+cargo install --locked ballista-executor
 ```
 
 With these crates installed, it is now possible to start a scheduler process.

--- a/docs/source/user-guide/distributed/deployment/cargo-install.md
+++ b/docs/source/user-guide/distributed/deployment/cargo-install.md
@@ -23,8 +23,8 @@ A simple way to start a local cluster for testing purposes is to use cargo to in
 the scheduler and executor crates.
 
 ```bash
-cargo install ballista-scheduler
-cargo install ballista-executor
+cargo install --locked ballista-scheduler
+cargo install --locked ballista-executor
 ```
 
 With these crates installed, it is now possible to start a scheduler process.

